hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -188,6 +191,11 @@ private void setDBOptionsProps(ManagedDBOptions dbOptions) {
     if (maxNumberOfOpenFiles != null) {
       dbOptions.setMaxOpenFiles(maxNumberOfOpenFiles);
     }
+    if (!rocksDbStat.equals(OZONE_METADATA_STORE_ROCKSDB_STATISTICS_OFF)) {
+      statistics = new ManagedStatistics();
+      statistics.setStatsLevel(StatsLevel.valueOf(rocksDbStat));
+      dbOptions.setStatistics(statistics);
+    }
```

Comment:
nit: `ManagedStatistics statistics` could be a local variable in `build()`.

I don't think it makes any difference functionally, but it would be easier to follow its lifecycle.  With instance variable, the reader needs to consider what happens if the same builder is used to build multiple `RDBStore` instances.

Reply:
Thanks ! I will address the comments later if no other suggestion

Reply:
The global variable `statistics` are easier to pass to the `RDBStore` In current method structure, , nothing else.